PDFOptimizer/Release_Notes.html

```diff
@@ -18,6 +18,20 @@
         <h1>Release Notes, Datalogics PDF Optimizer</h1>
   
 	<div class="entry-content">
+<p><strong>Version 3.3.0</strong> (August 8, 2023)<br/>
+<p>New Features:</p>
+<ul>
+  <li>Added PDF/X-6 Output Support</li>
+  <li>Added support for a Linux ARM version of PDF Optimizer</li>
+</ul>
+<p>Problem Corrections:</p>
+<ul>
+  <li>Fixed a crash generating reports for certain input</li>
+  <li>SF#45283 Web Form: Keep Pantone-Colors when color-conversion is 'on'</li>
+  <li>SF#45755 Getting a Segmentation fault when using Customer's JSON Profile</li>
+  <li>Running compression in optimizer 3.2 with the following files results in an output file without images</li>
```

Comment:
The jira descriptions don't make much sense the way they're written, I'd suggest instead using their associated release notes.